src/lib/ndpi_content_match.c.inc

```diff
@@ -1478,6 +1485,8 @@ static ndpi_tls_cert_name_match tls_certificate_match [] = {
   { "CN=cloudflareclient.com",         NDPI_PROTOCOL_CLOUDFLARE_WARP },
   { "O=V Kontakte LLC",                NDPI_PROTOCOL_VK },
   { "O=Yandex LLC",                    NDPI_PROTOCOL_YANDEX },
+  { "O=Riot Games, Inc.",              NDPI_PROTOCOL_RIOTGAMES   },
+  { "O=Riot Games Inc",                NDPI_PROTOCOL_RIOTGAMES   },
```

Comment:
Could you attach a trace matching these two rules, please?

Reply:
[certificate.zip](https://github.com/ntop/nDPI/files/11181242/certificate.zip)